packages/project/src/db/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { type Patch, applyPatches } from "immer";
+import { prisma } from "@webstudio-is/prisma-client";
+import { Styles } from "@webstudio-is/react-sdk";
+
+const loadStylesByTreeId = async (treeId: string) => {
```

Comment:
I used Tree["id"] so that in case we want to change that id at some point, we have all pointers referenced, I don't know if it makes much sense but it also reads quite well, because reader knows its not just any string, its the string used in the Tree id and can easily find the algorithm used for generating that id


I am pointing this out because its a general question of how we treat such strings  from types perspective, cc @istarkov  and @rpominov

Reply:
I also like to define as `Tree["id"]` for documentations purposes / to show intent. But no strong opinion